frontend/package.json

```
@@ -51,6 +51,7 @@
     "@zxcvbn-ts/language-it": "2.1.0",
     "assert": "2.0.0",
     "axios": "1.1.3",
+    "colorjs.io": "^0.4.1-patch.1",
```

Pin the dependency version
```suggestion
    "colorjs.io": "0.4.1-patch.1",
```
Also, really strange version number..? Are you sure that is the correct current stable version?

---

https://github.com/LeaVerou/color.js/releases/tag/v0.4.1-patch.1